### PR TITLE
Rework ledger integration 

### PR DESCRIPTION
**Pull request by tomlinton**
This adds a derivation path and account select step to the ledger wallet connect.

I also tried to simplify some of the fork stuff a bit, and remove stuff we don't use. I changed the fork network ID because it seems like MetaMask has removed the ability to set your own and it is stuck at 1337. 

I can't get the swap estimate stuff to provide an estimate for ledger but I'm not really sure why. Maybe @sparrowDom could take a look? It just sits on loading indefinitely. I'm hoping it is something to do with the fork.

### CODE REVIEW

**Comment by tomlinton** [comment]
So the swap worked for me on real mainnet so I guess it is just a fork issue.

**Review by franckc — APPROVED** [review]
Great! LGTM

**Review by sparrowDom — COMMENTED** [review]
The estimation stuff should now be working in local mode. Thanks for the massive code cleanup, Ledger integration improvements and adding the disconnect button. 

Left some code nitpicks inline. 

One UX nitpick: would be cool to say `loading` here until the accounts load:
Screenshot 2021-11-30 at 00 13 31
 
Seems that Metamask behaviour on local fork is a bit sketchy. If I reload the dapp and have Metamask set to forked mode `web3react` somehow assumes the `chainId` to be 1 instead of 1337. Which results in a wrong network:
Screenshot 2021-11-30 at 01 30 50
 
Cycling to Mainnet and back to localhost fixes the issue.

**Comment by sparrowDom** [comment]
This might be out of scope of this PR, maybe we can make a `good first issue` out of it. I would love it if our dapp would show an aggregated (dollar?) value of ether + 3 stablecoins for the first 5 accounts in each derivation path next to the option. 

Screenshot 2021-11-30 at 00 26 58
 
I would show the options immediately and show a loading indicator to the right of each option, until the balances are not fetched.

File: dapp/src/components/buySell/AddOUSDModal.js [hunk 1]
```diff
@@ -2,21 +2,28 @@ import React, { useState, useEffect } from 'react'
...
 const AddOUSDModal = ({ onClose, isMobile }) => {
-  const connectorIcon = useStoreState(AccountStore, (s) => s.connectorIcon)
   const ousdAddress = useStoreState(
     ContractStore,
     (s) => s.contracts && s.contracts.ousd.address
   )
   const provider = providerName()
   const [addressCopied, setAddressCopied] = useState(false)
+  const connectorName = useStoreState(AccountStore, (s) => s.connectorName)
+  const connectorIcon = get(
```
[the remark this answers is not in the document]
Comment (tomlinton) [reply dapp/src/components/buySell/AddOUSDModal.js]: 👍

**Comment by tomlinton** [comment]
Added a loading indicator while it loads ledger addresses and ETH balances. I think thats probably enough for now, we can expand it with other ideas later.

**Comment by sparrowDom** [comment]
Nice much nicer UX

**Pull request merged**